<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>includeExam</class>
 <widget class="QWidget" name="includeExam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>346</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>262</width>
    <height>346</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>262</width>
    <height>346</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="examDropbox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="courseDropbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="classLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Class:</string>
   </property>
  </widget>
  <widget class="QLabel" name="examLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Exam:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="questionTextbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Question:</string>
   </property>
  </widget>
  <widget class="QLabel" name="aLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>A)</string>
   </property>
  </widget>
  <widget class="QLabel" name="bLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>B)</string>
   </property>
  </widget>
  <widget class="QLabel" name="cLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>C)</string>
   </property>
  </widget>
  <widget class="QLabel" name="dLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>D)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerA">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerD">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>addButton</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>updateButton</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>removeButton</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newButton</sender>
   <signal>clicked()</signal>
   <receiver>newButton</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExam</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
